### PR TITLE
Add OnePlus N10 overlay

### PR DESCRIPTION
**Pull request by FinecoFinit**
Not tested! Just don't know how I can add it to /system/product/overlay . said "no space" Got it from [here](https://forum.xda-developers.com/t/flashing-gsi-roms-onto-your-nord-n10-5g.4294113/post-85212135) then added some values from values-land and xml power from latest official rom

### CODE REVIEW

**Comment by FinecoFinit** [comment]
Tested with Magisk module, it's working! But status margin in notifications menu is still not fixed(too narrow to camera cutout) and auto brightness now working

**Comment by FinecoFinit** [comment]
Any idea how I can push date with clock a little bit?
image

**Comment by FinecoFinit** [comment]
Same issue goes for lock screen statusbar

File: OnePlus/N10/res/xml/power_profile.xml [hunk 1]
```diff
@@ -12,105 +12,72 @@
...
-    </array>
-    <item name="modem.controller.voltage">0</item>
-    <array name="gps.signalqualitybased">
-        <value>0</value>
-        <value>0</value>
+        <value>1.6</value>
+        <value>1.6</value>
     </array>
-    <item name="gps.voltage">0</item>
+    <allow-in-power-save package="com.google.android.gms" />
```
Comment (phhusson) [inline OnePlus/N10/res/xml/power_profile.xml]: Uh, that doesn't belong in that file

**Comment by FinecoFinit** [comment]
@phhusson I think this can be merged

**Pull request merged**